e2e/cypress/integration/ui-smoke/api-endpoints.spec.js

```diff
@@ -0,0 +1,142 @@
+import '../../support/commands';
+import { v4 as uuidv4 } from 'uuid';
+
+describe('API endpoints work as expected', () => {
+  before(() => {
+    cy.login();
+  });
+
+  const { apiUrl } = Cypress.config();
+  const experimentId = '38eacf24-3f53-45ab-997d-36df0944f05d';
```

Comment:
Why this experiment? what experiment is this?

Reply:
Just a random experimentid I chose - has to follow the uuid pattern so I chose to be a realistic one